rtvi-client-js/src/core.ts

```diff
@@ -492,7 +493,7 @@ export abstract class Client extends (EventEmitter as new () => TypedEmitter<Voi
    * @param config - VoiceClientConfigOption[] partial object with the new configuration
    * @returns VoiceClientConfigOption[] - Updated configuration
    */
-  protected set config(config: VoiceClientConfigOption[]) {
+  public set config(config: VoiceClientConfigOption[]) {
```

Comment:
This one, I was receiving a warning, regarding different access modifiers for the `get` and `set`. 

But that is probably an issue from my IDE, I was just looking at this PR, and it seems it is supposed to work without any kind of warning: 
https://github.com/microsoft/TypeScript/pull/42425

Set accessors may be less visible than their corresponding get. 

So we should remove this change for sure.